configuration/listeners.md

```diff
@@ -74,9 +74,27 @@ To enable the PROXY protocol for tcp listeners use `listener.tcp.proxy_protocol
 
 If client certificates are used you can set `listener.tcp.proxy_protocol_use_cn_as_username = on` which will overwrite the MQTT username set by the client with the common name from the client certificate before authentication and authorization is performed. 
 
+## SSL/TLS Support
+VerneMQ supports different Transport Layer Security (TLS) options, which allow for secure communication between MQTT clients and VerneMQ. 
 
-## Sample SSL Config
+TLS provides secure communication between devices by encrypting the data in transit, preventing unauthorized access and ensuring the integrity of the data. VerneMQ supports various TLS options, including the use of certificates, mutual authentication, Pre-Shared Keys and the ability to specify specific ciphersuites and TLS versions.
 
+VerneMQ supports the following the TLS-flavours:
+
+- Server Side TLS
+- TLS-PSK
+- Mutal TLS (mTLS)
+
+In server-side TLS, the client initiates a TLS handshake with the broker, and the broker responds by sending its certificate. The client verifies the certificate and generates a symmetric key, which is used to encrypt and decrypt data exchanged between the client and broker. The broker does no further authentication or authorization of the client.  
```

Comment:
The broker *does* further authentication and authorization of the client, on the MQTT level!

Reply:
Added that as a clarification.